app/controllers/api/v1/notes_controller.rb

```diff
@@ -13,8 +13,22 @@ def show
         render json: note, status: :ok, serializer: NoteSerializer
       end
 
+      def create
+        return render_missing_params if missing_params
+        return render_invalid_type if invalid_note_type
```

Comment:
bieeen, solo un pequeñito detalle 👇🏻 los métodos que responden un boolean los nombramos con `?`
```suggestion
        return render_missing_params if missing_params?
        return render_invalid_type if invalid_note_type?
```

Reply:
Perfectoo